app/assets/scripts/config/production.js

```diff
@@ -6,7 +6,6 @@ var logo = require('./logo');
 module.exports = {
   environment: 'production',
   consoleMessage: logo,
-  api: 'https://api.openaq.org/v1',
+  api: 'https://api.openaq-staging.org/v1/upload',
```

Comment:
The api should not have the endpoint. just the root address. `https://api.openaq.org/v1`
Also, it shouldn't point to staging.